diff --git a/Cargo.toml b/Cargo.toml
index 4110567..dec319b 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -119,8 +119,7 @@ ci = ["github"]
 # The installers to generate for each app
 installers = []
 # Target platforms to build apps for (Rust target-triple syntax)
-targets = ["x86_64-unknown-linux-gnu", "x86_64-pc-windows-msvc"]
-unix-archive = ".zip"
+targets = ["x86_64-unknown-linux-gnu"]

 [dev-dependencies]
 mockall = "0.13.1"
